src/Umbraco.Cms.Integrations.Automation.Zapier/Controllers/ContentController.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+
+using System.Linq;
+
+using Umbraco.Cms.Integrations.Automation.Zapier.Configuration;
+using Umbraco.Cms.Integrations.Automation.Zapier.Models.Dtos;
+using Umbraco.Cms.Integrations.Automation.Zapier.Services;
+
+#if NETCOREAPP
+using Microsoft.Extensions.Options;
+
+using Umbraco.Cms.Web.Common.Controllers;
+using Umbraco.Cms.Core.Services;
+#else
+using System.Configuration;
+
+using Umbraco.Web.WebApi;
+using Umbraco.Core.Services;
+#endif
+
+namespace Umbraco.Cms.Integrations.Automation.Zapier.Controllers
+{
+    public class ContentController : UmbracoApiController
```

Comment:
Just to help understand the flow, think we could do with some XML header comments on this and the other various controllers.  I.e. for this one, explaining that it exposes and end-point to Zapier so it can create a dynamic drop-down list for the content types to pick from.

Reply:
Updated.